frontend/components/site/SiteUsers.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { SITE } from '../../propTypes';
+
+
+const SiteUsers = ({ site }) => {
+  // sort users by lower-cased usernames
+  const users = site.users.slice().sort((a, b) => {
+    const aName = a.username.toLowerCase();
+    const bName = b.username.toLowerCase();
+    if (aName === bName) { return 0; }
+    if (aName > bName) { return 1; }
+    return -1;
+  });
+
+  return (
+    <div>
+      <h4 className="label">Federalist users associated with this site</h4>
+      <table className="usa-table-borderless">
```

Comment:
Even though it currently only has one column, I made it a table in anticipation of having some kind of "remove user" action user in the near future.